src/components/notifications/Message.js

```diff
@@ -46,10 +49,13 @@ function AnalysisLink(props) {
     const { notification } = props;
     const { t } = useTranslation("common");
     const message = getDisplayMessage(notification);
-    const action = notification.payload?.action;
+    const payload = notification.payload;
```

Comment:
Since we know this notification's payload is an analysis at this point, I would refactor this to:
```suggestion
    const analysis = notification.payload;
```
I think that would make more sense further below as well (`analysis?.status === analysisStatus.COMPLETED` rather than `payload?.status === analysisStatus.COMPLETED` etc).